lightning/src/util/events.rs

```diff
@@ -338,7 +340,26 @@ pub enum Event {
 		channel_id: [u8; 32],
 		/// The full transaction received from the user
 		transaction: Transaction
-	}
+	},
+	/// Indicates that a path for an outbound payment was successful.
```

Comment:
Maybe mention that this is always generated after `PaymentSuccessfull`? I can otherwise read this as something that will be generated when one part of an MPP payment reaches the target even if other parts haven't yet (I know the protocol doesn't support doing that, but still).